containers/unit_tests/TestDualView.hpp

```diff
@@ -622,6 +622,42 @@ TEST(TEST_CATEGORY,
             const_dv.view<Kokkos::DefaultExecutionSpace>());
 }
 
+// User-defined types with a View data member, only host-constructible
+template <class V>
+class S {
+  V v_;
+
+ public:
+  template <class... Extents>
+  S(std::string label, Extents... extents) : v_(std::move(label), extents...) {}
+  S() : v_("v", 10) {}
+};
+
+template <typename V>
+auto test_sequential_host_init() {
```

Comment:
This function is named `test_sequential_host_init` but doesn't have any assertions or tests, maybe rename to `initialize_dual_view_sequential_host_init`